### PR TITLE
Added argsort frontend function to jax.numpy.namespace

### LINKED ISSUE #5832: argsort

### PR DESCRIPTION
**Pull request by ukj02**
Close  #5832

### CODE REVIEW

**Comment by ukj02** [comment]
I was not sure about my test implementation

**Comment by modanesh** [comment]
Maybe taking a look at Ivy tests doc could help: https://lets-unify.ai/ivy/deep_dive/ivy_tests.html

**Comment by modanesh** [comment]
I noticed that most of the `jax.numpy` namespace functions tests are failing with the code base. Could you update your PR to be based on this commit: https://github.com/unifyai/ivy/commit/697c634579c5b41552786a280060b6122fcbab91 or a later one so that we could review the test `jax.numpy` easier and with more confidence?

**Comment by modanesh** [comment]
Could you please fix the lint tests which failed relevant to your function and also check why `test-frontend-jax` test is failing? In addition, your commits still have conflicts with the master.

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ivy-seed** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.

**Comment by ivy-seed** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.
